cse-lib/data/src/test/resources/com/farao_community/farao/cse/data/ttc_rao/expected-simple-secure-cse-rao-result.xml

```diff
@@ -2,4 +2,103 @@
 <CseRaoResult>
     <Time v="2022-05-06T16:30Z"/>
     <Status v="SECURE"/>
+    <Results>
+        <PreventiveResult>
+            <PreventiveActions>
+                <Action name="complexNetworkActionId"/>
+                <Action name="pstRange1Id">
+                    <PSTtap v="-7"/>
+                </Action>
+            </PreventiveActions>
+            <MonitoredElement name="cnec1prevId">
+                <FromNode v="FR"/>
+                <ToNode v="FR"/>
+                <Code v="FFR1AA1  FFR3AA1  1"/>
```

Comment:
Values of these tags are wrong, it should be:
<FromNode v="FFR1AA1"/>
<ToNode v="FFR3AA1"/>
<Code v="1"/>